vac/raw/mix.md

```diff
@@ -889,9 +910,442 @@ are fixed strings and MUST be agreed upon across implementations.
+  &= 3984\ bytes
+\end{aligned}
+`$
+
+Implementations MUST account for payload extensions, such as SURBs,
+when determining the maximum message size that can be encapsulated in a
+single Sphinx packet. Details on SURBs are defined in
+[Section X.X].
+
+The following subsection defines the padding and fragmentation requirements for
+ensuring this fixed-size constraint.
+
+#### 8.3.3 Padding and Fragmentation
+
+Implementations MUST ensure that all messages shorter than the maximum payload size
+are padded before Sphinx encapsulation to ensure that all packets are
+indistinguishable on the wire. Messages larger than the maximum payload size MUST
+be fragmented by the origin protocol or top-level application before being passed
+to the Mix Protocol. Reassembly is the responsibility of the consuming application,
+not the Mix Protocol.
+
+#### 8.3.4 Anonymity Set Considerations
+
+The fixed maximum packet size is a configurable parameter. Protocols or
+applications that choose to configure a different packet size (either larger or
+smaller than the default) MUST be aware that using unique or uncommon packet sizes
+can reduce their effective anonymity set to only other users of the same size.
+Implementers SHOULD align with widely used defaults to maximize anonymity set size.
+
+Similarly, parameters such as $r$ and $t$ are configurable. Changes to these
+parameters affect header size and therefore impact payload size if the total packet
+size remains fixed. However, if such changes alter the total packet size on the
+wire, the same anonymity set considerations apply.
+
+The following subsection defines how the next-hop or destination address and
+forwarding delay are encoded within $β$ to enable correct routing and mixing
+behavior.
+
+### 8.4 Address and Delay Encoding
+
+Each hop’s $β$ includes a fixed-size block containing the next-hop address and
+the forwarding delay, except for the final hop, which encodes the destination
+address and a delay-sized zero padding. This section defines the structure and
+encoding of that block.
+
+The combined address and delay block MUST be exactly $tκ$ bytes in length,
+as defined in [Section 8.3.1](#831-header-field-sizes), regardless of the
+actual address or delay values. The first $(tκ - 2)$ bytes MUST encode the
+address, and the final $2$ bytes MUST encode the forwarding delay.
+This fixed-length encoding ensures that packets remain indistinguishable on
+the wire and prevents correlation attacks based on routing metadata structure.
+
+Implementations MAY use any address and delay encoding format agreed upon
+by all participating mix nodes, as long as the combined length is exactly
+$tκ$ bytes. The encoding format MUST be interpreted consistently by all
+nodes within a deployment.
+
+For interoperability, a recommended default encoding format involves:
+
+- Encoding the next-hop or destination libp2p multi-address:
+  - Each mix node is limited to one IPv4 circuit relay multiaddress to keep the
```

Comment:
Does this clause mean that the _address_ is limited in _size_ to accommodate for one circuit relay address? If my understanding is correct, perhaps it could be clearer to first state the "encoding layout" as `multiaddr` and then explain the (size) limit in terms of what granularity can be encoded?